.github/workflows/release.yml

```diff
@@ -37,7 +37,7 @@ jobs:
       - name: Build
         run: dotnet build src --configuration Release -graph --no-restore
       - name: Build ServiceControl Management
-        run: dotnet publish src\ServiceControl.Config\ServiceControl.Config.csproj --configuration Release --runtime win-x64 --no-build --output assets
+        run: dotnet publish src\ServiceControl.Config\ServiceControl.Config.csproj --no-build --output assets
```

Comment:
Release is unnecessary with no-build I guess?

Reply:
The .NET 8 SDK changed `dotnet publish` to default to `Release`, so there's no longer a reason to specify it.